hw/xfree86/common/xf86DGA.c

```diff
@@ -1783,17 +1754,18 @@ ProcXF86DGAGetVideoLL(ClientPtr client)
     /* get the parameters for the mode that best matches */
     DGAGetModeInfo(stuff->screen, &mode, num);
 
+    xXF86DGAGetVideoLLReply rep = {
+        .offset = mode.offset,
+        .width = mode.bytesPerScanline / (mode.bitsPerPixel >> 3),
+        .ram_size = rep.bank_size >> 10
```

Comment:
Using `rep.bank_size` here looks wrong.

Reply:
ups, you're right. in the original, DGAOpenFramebuffer() directly wrote into some reply struct fields (w/ ugly hard typecasts), so overseen this.